community/tcl/build

```diff
@@ -1,14 +1,21 @@
 #!/bin/sh -e
 
-IFS=. read -r ver1 ver2 _ < "${0%/*}/version"
+IFS=. read -r maj min _ < "${0%/*}/version"
 
-unix/configure \
+rm -rf pkgs/sqlite3*
+
+cd unix
+
+./configure \
```

Comment:
`unix/configure`? Why change it?

Reply:
avoids having to specify `unix/...` multiple times as per the changes in this PR.

Reply:
I only see `unix/...` being used once for `configure`?

Reply:
it'd also have to be specified on the last line (if not for the `cd`)

Reply:
I went with the cd because my thinking is there is also a win/ and a macos/ directory each with their own makefiles as well, and to cd explicitly to the unix/ directory and work entirely from it seemed a safe choice.

Reply:
There's no need to `cd`.